tools/devtools/src/page_scripts/owl_devtools_global_hook.js

```diff
@@ -51,6 +51,12 @@
       this.traceSubscriptions = false;
       this.requestedFrame = false;
       this.eventsBatch = [];
+      // This is for retrocompatibility purposes since new versions of owl should always expose toRaw and reactive
+      // in __OWL_DEVTOOLS__
+      this.toRaw = window?.owl?.toRaw ? window.owl.toRaw : window.__OWL_DEVTOOLS__?.toRaw;
+      this.reactive = window?.owl?.reactive
+        ? window.owl.reactive
+        : window.__OWL_DEVTOOLS__?.reactive;
```

Comment:
```suggestion
      this.toRaw = window.__OWL_DEVTOOLS__.toRaw ?? window.owl?.toRaw;
      this.reactive = window.__OWL_DEVTOOLS__.reactive ?? window.owl?.reactive;
```
`window.__OWL_DEVTOOLS__` is known to exist (it's used a little higher). This code should probably be located with the other code that extracts stuff from `__OWL_DEVTOOLS__`